src/Aspire.Hosting.Garnet/GarnetPersistenceCommandLineArgsCallbackAnnotation.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.Garnet;
+
+internal class GarnetPersistenceCommandLineArgsCallbackAnnotation(TimeSpan interval, long keysChangedThreshold)
+    : CommandLineArgsCallbackAnnotation(context =>
+    {
+        context.Args.Add("--save");
+        context.Args.Add(interval.TotalSeconds.ToString(CultureInfo.InvariantCulture));
+        context.Args.Add(keysChangedThreshold.ToString(CultureInfo.InvariantCulture));
+        return Task.CompletedTask;
+    })
+{
+    public TimeSpan Interval { get; } = interval;
+
+    public long KeysChangedThreshold { get; } = keysChangedThreshold;
```

Comment:
Why do we need these properties? Is it only for tests?

Maybe the tests can assert the command line args that are added instead.

Reply:
Unfortunately, I didn't quite understand what to do with this

Reply:
I guess I would remove them then.

Reply:
Deleted

I also translated the tests for verification via command line args